src/runtime/utils.ts

```diff
@@ -81,10 +81,44 @@ export async function loadFile(url: string): Promise<string> {
  */
 export class Markup extends String {}
 
+function _escapeHtml(str: any): string {
```

Comment:
I suppose this is done elsewhere in the code (see the code that compiles the t-set instruction).
Node.textContent is safe against XSS (https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) and is basically used in owl natively.

Reply:
```js
el = document.createElement("span")
// <span>

el.textContent = `<img src="lol" onerror="alert('xss')">`
// `<img src="lol" onerror="alert('xss')">`

el.innerHTML
// `&lt;img src="lol" onerror="alert('xss')"&gt;` 
```